modules/coredns/testdata/version170/some_load.txt

```diff
@@ -0,0 +1,180 @@
+# HELP coredns_build_info A metric with a constant '1' value labeled by version, revision, and goversion from which CoreDNS was built.
+# TYPE coredns_build_info gauge
+coredns_build_info{goversion="go1.14.4",revision="f59c03d",version="1.7.0"} 1
+# HELP coredns_panics_total A metrics that counts the number of panics.
+# TYPE coredns_panics_total counter
+coredns_panics_total 0
+# HELP coredns_dns_requests_total Counter of DNS requests made per zone, protocol and family.
+# TYPE coredns_dns_requests_total counter
+coredns_dns_requests_total{family="1",proto="udp",server="",zone="dropped"} 21
+coredns_dns_requests_total{family="1",proto="udp",server="dns://:53",zone="coredns.io."} 6
+coredns_dns_requests_total{family="1",proto="udp",server="dns://:53",zone="dropped"} 9
+coredns_dns_requests_total{family="1",proto="udp",server="dns://:53",zone="ya.ru."} 9
+coredns_dns_requests_total{family="1",proto="udp",server="dns://:54",zone="coredns.io."} 13
+coredns_dns_requests_total{family="1",proto="udp",server="dns://:54",zone="dropped"} 12
+coredns_dns_requests_total{family="1",proto="udp",server="dns://:54",zone="ya.ru."} 12
+# HELP coredns_dns_requests_total Counter of DNS requests per type, per zone.
+# TYPE coredns_dns_requests_total counter
+coredns_dns_requests_total{server="",type="A",zone="dropped"} 7
+coredns_dns_requests_total{server="",type="AAAA",zone="dropped"} 7
```

Comment:
How did you get this test data? Using `sed`? :smile:

[It should be](https://github.com/coredns/coredns/tree/master/plugin/metrics#description)

- `coredns_dns_requests_total{server, zone, proto, family, type}`

And you have same metric name with different label set:
 - coredns_dns_requests_total{family, proto, server, zone}
 - coredns_dns_requests_total{server, type, zone}

Reply:
I actual tried to cheat and copy the data from v1.6.9 but it didn't work my way. Ok, now last commit should have valid test data.